packages/create-llama/helpers/types.ts

```diff
@@ -23,4 +23,5 @@ export interface InstallTemplateArgs {
   communityProjectPath?: string;
   vectorDb?: TemplateVectorDB;
   externalPort?: number;
+  isPoetryInstall?: boolean;
```

Comment:
generalize this question (also ask it for TS) and rename variable
```suggestion
  installDependencies?: boolean;
```